src/pages/recipe/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+import {  FullRecipe, Loading } from "../../components";
+import { useGetUserStars } from "../../Hooks";
+import { getUser } from "../../Functions";
+
+export const Recipe = () => {
+  const { state } = useLocation();
+  const recipe = state?.recipe;
+  const user = getUser();
+  const { isLoading, data } = useGetUserStars(recipe?._id, user?.id);
+  if (isLoading) return <Loading />;
+  return (
```

Comment:
Be consistent with the way you are showing the loading: 

This is one way you are doing it, and before you used something like `isloading ? <Loading/> : renderSomethingElse`